rsconf/systemd.py

```diff
@@ -71,9 +71,9 @@ def custom_unit_enable(
     unit_enable(compt, j2_ctx)
 
 
-def custom_unit_prepare(compt, j2_ctx, watch_files=()):
+def custom_unit_prepare(compt, j2_ctx, watch_files=(), run_d=None):
     """Must be first call"""
-    run_d = unit_run_d(j2_ctx, compt.name)
+    run_d = run_d or unit_run_d(j2_ctx, compt.name)
```

Comment:
Use a temporary or use `if not run_d: run_d = ...`.  I don't like assigning run_d if already assigned.

Reply:
fixing the compt.name removed the need for supplying run_d https://github.com/radiasoft/rsconf/commit/5d2406d37cba0a5b455f327bc6df141c05c6010a